packages/components/src/border-box-control/border-box-control-linked-button/component.tsx

```diff
@@ -25,18 +24,16 @@ const BorderBoxControlLinkedButton = (
 	const label = isLinked ? __( 'Unlink sides' ) : __( 'Link sides' );
 
 	return (
-		<Tooltip text={ label }>
-			<View className={ className }>
-				<Button
-					{ ...buttonProps }
-					size="small"
-					icon={ isLinked ? link : linkOff }
-					iconSize={ 24 }
-					aria-label={ label }
-					ref={ forwardedRef }
-				/>
-			</View>
-		</Tooltip>
+		<View className={ className }>
```

Comment:
Can we try to remove this View component and set the className directly on the Button? Not sure the View is necessary at all (it's just a div element that wraps the button).